### DISCUSSION TITLE
Inline footnotes for GFM

### DISCUSSION THREAD

**Discussion by timlrx** [remarkjs/remark#911] [General] [answered]
It looks like `remark-footnotes` is now tagged as legacy and the recommendation is to use the updated `remark-gfm` instead. However, `remark-gfm` does not support inline footnotes and it also does not seem to be supported as part of the GFM spec. Could we consider continue maintaining `remark-footnotes` or have a new `remark-inline-footnotes` plugin?

**Comment by wooorm** [comment, marked as answer]
You can use `remark-footnotes`, it’s not deprecated yet. You can also make an inline footnotes plugin yourself. I recommend not using inline footnotes. GH is giant and inline notes are not specified.

**Reply by timlrx** [reply]
Thanks, unfortunately I have too many existing markdown files using inline footnotes. Will just use `remark-footnotes` with `remark-gfm` for the time being. It seems to work for now.

**Reply by wooorm** [reply]
remark can format markdown. With a small plugin you could change inline notes to footnotes and automatically fix this
